lens.cabal

```diff
@@ -412,40 +401,17 @@ test-suite hunit
       test-framework       >= 0.6,
       test-framework-hunit >= 0.2
 
--- Verify the results of the examples
+-- We need this dummy test-suite to add simple-reflect to the install plan
 test-suite doctests
   type:              exitcode-stdio-1.0
   main-is:           doctests.hs
-  ghc-options:       -Wall -threaded
   hs-source-dirs:    tests
   default-language:  Haskell2010
-  x-doctest-options: --fast
```

Comment:
Does `--fast` have an equivalent in `cabal-docspec`? I remember this being a pain point for `doctest` at one point.

Reply:
That is discussed in the manual.

https://github.com/phadej/cabal-extras/blob/master/cabal-docspec/MANUAL.md#a-note-on-performance (looks like I miss a word there).

> Because cabal-docspec uses compiled library, calling :reload: after each group doesn't cause performance problems, reloading is fast. For that reason, cabal-docspec doesn't have --fast variant, it is not needed.

Reply:
The `docspec` step on GHA takes 40seconds. I think it is reasonable. (`doctest` with `--fast` took about the same time).

Reply:
That's what I get for not reading the freaking manual. Thanks for pointing that out!

Reply:
No worries. Not all your questions have answers there (yet).